src/main/java/co/inventorsoft/model/Person.java

```diff
@@ -16,4 +16,10 @@ public String getName() {
     public Integer getAge() {
         return age;
     }
+
+    public boolean isTeenager(){
+        if (this.getAge() < 19 && this.getAge() > 13)
+            return true;
+        return false;
```

Comment:
Cool, but can you try to do it in one statement?:)

Reply:
yes, of course)